src/style.css

```diff
@@ -0,0 +1,78 @@
+@import url(https://fonts.googleapis.com/css2?family=Josefin+Sans:ital,wght@0,100;0,400;0,700;1,400&family=Lato:ital,wght@0,400;0,700;0,900;1,900&family=Poppins:wght@400;700&family=Raleway:wght@300;900&display=swap);
+
+body {
+  font-family: "Josefin Sans", sans-serif;
+  font-style: normal;
+  background: rgb(242, 242, 242);
+}
+
+.wrapper {
+  font-style: normal;
+  padding-left: 40px;
+  display: flex;
+  justify-content: space-between;
+  width: 35%;
```

Comment:
- [ ] Your project currently doesn't match completely the provided wireframe as shown below. Your design is expected to span the entire viewport of the users screen but it is only occupying a small section as shown below. I kindly suggest you increase the width to achieve the project design 👍 ✨ 
- project wireframe
![wfi](https://github.com/ethred/Leaderboard/assets/89055502/f0aabcdb-9f9c-4329-bbf2-0a19f5267af7)
- your design
![wfu](https://github.com/ethred/Leaderboard/assets/89055502/c48f4a75-8418-4e74-9701-76f063400c91)